Packs/CommonTypes/IndicatorFields/indicatorfield-globalfirstseen.json

```diff
@@ -0,0 +1,36 @@
+{
+    "id": "indicator_globalfirstseen",
+    "version": -1,
+    "modified": "2024-10-30T12:05:32.803043623Z",
+    "name": "Global First Seen",
+    "ownerOnly": false,
+    "cliName": "globalfirstseen",
+    "type": "date",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 2,
+    "hidden": false,
+    "openEnded": false,
+    "description": "First time an indicator was detected globally across all organizations.",
```

Comment:
```suggestion
    "description": "First time the indicator was detected globally across all organizations.",
```

Reply:
As discussed in the demo, "Global First Seen" will not be created as we would use the existing "First Seen By Source" field instead.

_Resolving as this file has been deleted._